matrix-sdk-android/src/test/java/org/matrix/android/sdk/internal/database/migration/MigrateSessionTo033Test.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2022 The Matrix.org Foundation C.I.C.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.matrix.android.sdk.internal.database.migration
+
+import org.junit.Test
+import org.matrix.android.sdk.internal.database.model.RoomSummaryEntityFields
+import org.matrix.android.sdk.test.fakes.FakeDynamicRealm
+
+internal class MigrateSessionTo033Test {
```

Comment:
I think tests for Realm migrations are excessive. Migration classes never change, so there is no need to retest them, but we put extra work on our CI to run this tests

Reply:
Tests removed